packages/flutter_tools/templates/ffi_plugin/pubspec.yaml.tmpl

```diff
@@ -0,0 +1,94 @@
+name: {{projectName}}
+description: {{description}}
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: {{dartSdkVersionBounds}}
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  ffigen: ^4.1.2
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.0
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+  # This section identifies this Flutter project as a plugin project.
+  # The 'pluginClass', ffiPlugin, and Android 'package' identifiers should not ordinarily
+  # be modified. They are used by the tooling to maintain consistency when
```

Comment:
Rather than saying they shouldn't be modified we should say what they mean and what the usual way to update them is. Telling developers that something is off limits... never works.